modules/cli/src/main/scala/scala/cli/commands/pgp/PgpScalaSigningOptions.scala

```diff
@@ -10,16 +10,18 @@ import scala.cli.commands.tags
 final case class PgpScalaSigningOptions(
   @Group(HelpGroup.Signing.toString)
   @Tag(tags.restricted)
+  @HelpMessage("Version of scala-cli-signing to use when running it externally")
   @Hidden
     signingCliVersion: Option[String] = None,
   @Group(HelpGroup.Signing.toString)
   @Tag(tags.restricted)
+  @HelpMessage("Pass arguments to the java command when running scala-cli-singing externally on JVM")
   @ValueDescription("option")
   @Hidden
     signingCliJavaArg: List[String] = Nil,
   @Group(HelpGroup.Signing.toString)
   @Tag(tags.restricted)
-  @HelpMessage("Whether to run the Scala Signing CLI on the JVM or using a native executable")
+  @HelpMessage("When using externally run scala-cli-singing, force the use of JVM to execute it")
   @Hidden
     forceJvmSigningCli: Option[Boolean] = None
```

Comment:
What is the difference between options `forceJvmSigningCli` and  `forceSigningExternally` ? Is it the same?